src/pages/Categories.jsx

```diff
@@ -19,11 +21,16 @@ const Categories = () => {
   return (
     <section className="main-page">
       <h2 className="page-heading">Categories</h2>
-      <ul className="category-list">
+      <ul className="main-list">
         {categoryList.map((categoryItem) => {
           return (
-            <li key={categoryItem.slug}>
+            <li key={categoryItem.slug} className="main-card">
               <CategoryCard category={categoryItem} />
+              <Link to={`/reviews?category=${categoryItem.slug}`}>
+                <button onClick={() => setCategoryName(categoryItem.slug)}>
+                  See all reviews for {categoryItem.slug}
+                </button>
+              </Link>
```

Comment:
There isn't a route for this, so it doesnt work when you manually type in the url. It just loads them all